### PR TITLE
Fix wording in one sentence in getting-started.html

### PR DESCRIPTION
**Pull request by mr-nati**
Fix wording in one sentence in getting-started.html

### CODE REVIEW

File: website/static/getting-started.html [hunk 1]
```diff
@@ -242,8 +242,8 @@ <h3 id="reactive-reactjs-components">Making React reactive</h3>
                         <p>Ok, so far we made a silly report reactive. Time to build a reactive user interface around this very
                             same store. React components are (despite their name) not reactive out of the box.
                             The <code>observer</code> HoC wrapper from the <code>mobx-react-lite</code> package fixes that by
-                            basically wrapping the React component in <code>autorun</code>. To automatically
-                            keep components in sync with the state. This is conceptually not different from what we did
+                            basically wrapping the React component in <code>autorun</code>. This automatically
+                            keep the component in sync with the state. This is conceptually not different from what we did
                             with the <code>report</code> before.
```
Comment (kubk) [inline website/static/getting-started.html:247]: @mr-nati Should `keep` be changed to `keeps` also? I mean "This automatically keep**s** the component in sync with the state"

Comment (urugator) [reply website/static/getting-started.html:247]: and remove `automatically`:

_This keeps the component in sync with the state._

Comment (mr-nati) [reply website/static/getting-started.html:247]: Thanks!

**Comment by kubk** [comment]
@mr-nati Thank you 👍

**Pull request merged**